gretel/gc-nlp_text_analysis/manifest.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Working Safely with Sensitive Free Text Using Gretel.ai and NLP",
```

```suggestion
    "name": "Work Safely with Sensitive Free Text Using Gretel and NLP",
```

Suggesting the change `Working` -> `Work` to follow the semantic language of the other blueprints which use primarily verbs.

---

Fixed. I also dropped the NLP callout and moved it to the description.

> Work Safely with Sensitive Free Text Using Gretel